package.json

```diff
@@ -206,7 +206,7 @@
   },
   "dependencies": {
     "@babel/runtime": "^7.18.9",
-    "@blockaid/ppom": "^1.1.1",
+    "@blockaid/ppom": "^1.1.2",
```

Comment:
I'm slightly confused by the description. 
```
1. Revert ppom.js to previous version as that is compatible with the @blockaid/ppom 1.1.1.
```

should we update the version to `^1.1.2` since we are updating the @blockaid/ppom version?
maybe we can include this in the title as well 

`Reverting ppom.js file to old version` → `Revert ppom.js file to old version and bump @blockaid/ppom ^1.1.1 → ^1.1.2`

Reply:
@digiwand : PR is old, it was made for @blockaid/ppom version 1.1.1, but now version in develop is new.

Reply:
ok, sounds like updating the PR description would be a good idea. can we update it to reflect what you've mentioned?

Reply:
Done @digiwand